diagnostics/networking.bat

```diff
@@ -0,0 +1,36 @@
+@echo off
+:: Note: This script terminates WSL. Save your work before running it.
+
+:: Check for administrator access
+net session >nul 2>&1 || goto :admin
+
+:: Validate that required files are here
+if not exist wsl.wprp (echo wsl.wprp not found && exit /b 1)
+if not exist networking.sh (echo networking.sh not found && exit /b 1)
+
+
+:: Stop WSL
+sc stop LxssManager
+
+:: List all HNS objects
+echo HNS objects: 
+hnsdiag list all -df
+
+:: The WSL HNS network is created once per boot. Resetting it to collect network creation logs
+echo Deleting HNS network
+hnsdiag reset networks
```

Comment:
This would reset all the networks according to hnsdiag. Why not just delete the wsl ics network only? It has a fixed name iirc

Reply:
Good point. Changed the script to capture the correct network GUID.